src/components/page-step/job-step-logs.tsx

```diff
@@ -0,0 +1,63 @@
+import * as PropTypes from 'prop-types';
+import AsyncResource from '../async-resource/simple-async-resource';
+import './style.css';
+import { usePollJobStepLogs } from './use-poll-job-step-logs';
+import { useGetJobStepFullLogs } from './use-get-job-step-full-logs';
+import { Log, LogDownloadOverrideType } from '../component/log';
+import { Accordion, Typography } from '@equinor/eds-core-react';
+
+export interface StepLogsProps {
+  appName: string;
+  jobName: string;
+  stepName: string;
+}
+
+export const JobStepLogs = ({
+  appName,
+  jobName,
+  stepName,
+}: StepLogsProps): JSX.Element => {
+  const [pollStepLogsState] = usePollJobStepLogs(appName, jobName, stepName);
+  const [getStepFullLogsState, downloadFullLog] = useGetJobStepFullLogs(
+    appName,
+    jobName,
+    stepName
+  );
+  const downloadOverride: LogDownloadOverrideType = {
+    status: getStepFullLogsState.status,
+    content: getStepFullLogsState.data,
+    onDownload: () => downloadFullLog(),
+    error: getStepFullLogsState.error,
+  };
+
+  return (
+    <section className="step-log">
+      <AsyncResource asyncState={pollStepLogsState}>
+        {pollStepLogsState?.data ? (
+          <Accordion className="accordion elevated" chevronPosition="right">
+            <Accordion.Item isExpanded>
+              <Accordion.Header>
+                <Typography variant="h4">Log</Typography>
+              </Accordion.Header>
+              <Accordion.Panel>
+                <Log
+                  downloadOverride={downloadOverride}
+                  fileName={jobName + stepName + '.txt'}
+                  logContent={pollStepLogsState.data}
+                ></Log>
```

Comment:
Elements without children can be shortened

```suggestion
                <Log
                  downloadOverride={downloadOverride}
                  fileName={jobName + stepName + '.txt'}
                  logContent={pollStepLogsState.data}
                />
```